gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/language/translator/GremlinTranslatorTest.java

```diff
@@ -741,24 +741,51 @@ public static Collection<Object[]> data() {
                             null,
                             null,
                             "g.inject(long(1694017707000)).as_date()"},
-                    {"g.inject(\"123.4\").asNumber(N.int)",
-                            null,
-                            "g.inject(string0).asNumber(N.int)",
-                            "g.Inject<object>(\"123.4\").AsNumber(N.Int)",
-                            "g.Inject(\"123.4\").AsNumber(gremlingo.N.Int)",
-                            "g.inject(\"123.4\").asNumber(N.int_)",
-                            "g.inject(\"123.4\").asNumber(N.int_)",
-                            "g.inject(\"123.4\").asNumber(N.int_)",
-                            "g.inject('123.4').as_number(N.int)"},
-                    {"g.inject(\"123.4\").asNumber(N.bigInt)",
-                            null,
-                            "g.inject(string0).asNumber(N.bigInt)",
-                            "g.Inject<object>(\"123.4\").AsNumber(N.BigInt)",
-                            "g.Inject(\"123.4\").AsNumber(gremlingo.N.BigInt)",
-                            "g.inject(\"123.4\").asNumber(N.bigInt)",
-                            "g.inject(\"123.4\").asNumber(N.bigInt)",
-                            "g.inject(\"123.4\").asNumber(N.bigInt)",
-                            "g.inject('123.4').as_number(N.big_int)"},
+                    {"g.inject(\"123.4\").asNumber(GType.INT)",
+                            null,
+                            "g.inject(string0).asNumber(GType.INT)",
+                            "g.Inject<object>(\"123.4\").AsNumber(GType.Int)",
+                            "g.Inject(\"123.4\").AsNumber(gremlingo.GType.Int)",
+                            "g.inject(\"123.4\").asNumber(GType.INT)",
+                            "g.inject(\"123.4\").asNumber(GType.INT)",
+                            "g.inject(\"123.4\").asNumber(GType.int)",
+                            "g.inject('123.4').as_number(GType.INT)"},
+                    {"g.inject(\"123.4\").asNumber(GType.BIGINT)",
```

Comment:
Is there a reason the Java and Groovy cases are excluded from most of these scenarios? Also I think it would be good to have a few which use some of the lowercase GType tokens in addition to all of these using the all caps forms.